docs/usage-cli.rst

```diff
@@ -75,7 +75,7 @@ See below how sample configuration file could look like. The structure is the sa
    blueprint: api-description.apib
    endpoint: "http://127.0.0.1:3000"
 
-.. note::
+.. warning::
```

Comment:
I might have confused you by pointing at random things on your display and saying what should be warning 😄 I think this should stay as `note`

Reply:
Resolved. Thanks :)